pkg/arvo/app/graph-push-hook.hoon

```diff
@@ -32,6 +32,22 @@
   ?|  (is-member:grp src.bowl i.group-paths)
       (is-admin:grp src.bowl i.group-paths)
   ==
+::
+++  is-allowed-remove
+  |=  [=resource:res indices=(set index:store) =bowl:gall]
+  ^-  ?
+  =/  gra   ~(. graph bowl)
+  ?.  (is-allowed resource bowl %.n)
+    %.n
+  %+  levy
```

Comment:
What does this do? The method description in the standard library is... not helpful.

Reply:
Oh, okay. It just returns yes if all of the result tests are yes... essentially an ?& across a list.